### PR TITLE
updated to consume ember-frost-modal

### PR DESCRIPTION
**Pull request by quincyle**
This is a #major# change.

# CHANGELOG
**Updated**  to consume `ember-frost-modal`.
**Removed** bunsen binding in addon. Consumer now can add bunsen in 'body' slot section.

### CODE REVIEW

**Comment by rox163** [comment]
This will be #Major because we are removing bunsen from the internal component and just making it a yield.

**Comment by sandersky** [comment]
@rox163 What purpose will this addon provide that `ember-frost-modal` doesn't?

**Comment by rox163** [comment]
this is supposed to provide styling for form content specifically. 
modal-dialog provides styling for info/warning dialog type content.

**Comment by sglanzer** [comment]
@sandersky I think the primary difference is the entry/exit animation and the position on-screen of input versus messages.  Whether that warrants a second component at this stage I don't know, probably makes more sense as an second component in the modal repo, but we can roll with this for now and clean up later

File: app/templates/components/frost-modal-input.hbs [hunk 1]
```diff
@@ -26,34 +21,25 @@
...
-        button=(component 'frost-button'
-        size='medium'
-        vertical=true)))}}
-    </div>
-  {{/yield-slot}}
-{{/ember-remodal}}
+  {{#block-slot slot 'footer' as |controls|}}
+    {{controls.cancel}}
+    {{#yield-slot 'footer'}}
+      {{yield (block-params
```
Comment (rox163) [inline app/templates/components/frost-modal-input.hbs]: perfect!

**Comment by rox163** [comment]
:+1:

File: blueprints/ember-frost-modal-input/index.js [hunk 2]
```diff
@@ -14,8 +14,6 @@ module.exports = {
         {name: 'ember-frost-bunsen', target: '^6.0.3'},
         {name: 'ember-frost-core', target: '>=0.0.14 <2.0.0'},
```
Comment (rox163) [inline blueprints/ember-frost-modal-input/index.js]: can you updated the core version here?

**Pull request merged**